README.md

```diff
@@ -1,56 +1,24 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# HELLO RAILS
 
-> Description the project.
+> RUBY API For Random greetings
 
 
-## Built With
+## LINK TO REACT PROJECT PR
 
-- Major languages
-- Frameworks
-- Technologies used
```

Comment:
[Highly Recommended] [OPTIONAL]

- I would recommend making the README professional by adding all the proper steps to run the project locally. Also, update the  Frontend application README file so that users can easily run the whole application.

```
git clone https://github.com/peterdgreat/hello-rails-back-end.git
cd hello-rails-back-end
bundle

# Setup the database and run the server
rails db:create
rails db:migrate
rails db:seed
rails s
```